modules/insights-operator-what-information-is-collected.adoc

```diff
@@ -0,0 +1,22 @@
+// Module included in the following assemblies:
+//
+// * support/insights-operator/about-insights-operator.adoc
+
+[id="insights-operator-what-information-is-collected_{context}"]
+= Information collected by the Insights Operator
+
+The Insights Operator collects the following information:
+
```

Comment:
Do we need punctuation after the bullet items? 
* The version of the cluster and its components;